Makefile

```diff
@@ -1,26 +1,28 @@
-.PHONY: help all clean install flake8 isort lint test
+.PHONY: help all clean install format test
 .DEFAULT_GOAL := help
 
 help: ## See what commands are available.
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36mmake %-15s\033[0m # %s\n", $$1, $$2}'
 
 
-all: install clean test lint ## Install, test and lint the project.
+all: clean install format test ## Install, format and test the project.
 
 clean: ## Remove Python file artifacts.
-	find . -type f -name "*.py[co]" -delete
-	find . -type d -name "__pycache__" -delete
+	find . -name '*.pyc' -exec rm -rf {} +
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.egg-info' -exec rm -rf {} +
+	rm -rf build dist
 
 install: ## Install dependencies.
 	pip install -e .[test]
 
-flake8: ## Run flake8 on the project.
-	flake8 src/
-
-isort: ## Run isort on the project.
-	isort --check-only --diff --recursive src/
-
-lint: flake8 isort ## Lint the project.
+format: ## Format the code.
+	black src/ tests/ examples/
+	isort src/ tests/ examples/
+	flake8 src/ tests/ examples/
```

Comment:
Is it necessary to pass arguments to these tools? I would expect their ignore rules to exclude non-source-code files.

Reply:
Consider it done 😄